### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by vadimdom**

### CODE REVIEW

File: __tests__/state-functions.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('Addition', () => {
+  it('knows that 2 and 2 make 4', () => {
+    expect(2 + 2).toBe(4);
+  });
+});
```
Comment (DinAlla) [inline __tests__/state-functions.test.js]: I think we should write "test" instead "it". Like this:
describe('...', () => {
  test('smth about', () => {
    //your tests
  });
});

Comment (DinAlla) [inline __tests__/state-functions.test.js]: I think we should write "test" instead "it". Like this:
describe('...', () => {
test('smth about', () => {
//your tests
});
});

File: __tests__/unit/state-functions.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('Addition', () => {
+  it('knows that 2 and 2 make 4', () => {
+    expect(2 + 2).toBe(4);
+  });
+});
```
Comment (DinAlla) [inline __tests__/unit/state-functions.test.js]: the same

File: __tests__/state-functions.test.js (hunk 1 above)
Comment (vadimdom) [reply __tests__/state-functions.test.js]: okay

**Pull request merged**